src/pages/dbot/index.tsx

```diff
@@ -91,6 +91,13 @@ const trading: TradingType[] = [
 
 const Dbot = () => {
     const { is_eu, is_row } = useRegion()
+    const [is_loaded, setLoaded] = useState(false)
+
+    useEffect(() => {
+        setLoaded(true)
+    }, [])
+
+    // if (is_loaded) {
```

Comment:
can we remove unnecessary comments?

Reply:
Hi @NikitK-deriv , Actually I made this PR after fixing some issue but those issues were persist on staging link. 
I couldn’t reproduce those issue on local so I was tried to debug on test link that's why I need to push again and again. 

I will definitely remove unnecessary comments before getting it merged. 😉 

Thanks man.